README.md

```diff
@@ -2,32 +2,19 @@
 
 ThreatExchange is a set of RESTful APIs on the Facebook Platform for querying, publishing, and sharing security threat information. It's a lightweight way for exchanging details on malware, phishing pages, and other threats with either specific members of the community or the ThreatExchange community at large.
```

Comment:
We should add a link to the public documentation very early on in the document.

https://developers.facebook.com/docs/threat-exchange

Reply:
Done